app/controllers/templates_controller.rb

```diff
@@ -109,8 +109,10 @@ def resource_name
 
   private
 
-  def safe_render(template)
-    render :plain => @host.render_template(template: template, params: params)
```

Comment:
😢 This method had a short life :-(
We should ideally keep this as unattended controller still uses the method.

Reply:
In addition, the reporting template should probably use it's own scope.
Maybe we need to use `template.get_scope` or something...

Reply:
my thoughts after our IRC discussion, see the latest commit, I couldn't resist :-) we can keep it for now or I can do the similar change there if you prefer